src/module/actor/data-model-classes/data-model-character-encumbrance.ts

```diff
@@ -88,13 +89,34 @@ export default class OseDataModelCharacterEncumbrance
   }
 
   get atHalfEncumbered() {
-    return this.value > this.max * (OseDataModelCharacterEncumbrance.encumbranceSteps.half / 100) + (this.#delta || 0)
+    return (
+      this.value >
+      this.max * (OseDataModelCharacterEncumbrance.encumbranceSteps.half / 100)
+    );
   }
   get atThreeEighthsEncumbered() {
-    return this.value > this.max * (OseDataModelCharacterEncumbrance.encumbranceSteps.threeEighths / 100) + (this.#delta || 0)
+    return (
+      this.value >
+      this.max *
+        (OseDataModelCharacterEncumbrance.encumbranceSteps.threeEighths / 100)
+    );
   }
   get atQuarterEncumbered() {
-    return this.value > this.max * (OseDataModelCharacterEncumbrance.encumbranceSteps.quarter / 100) + (this.#delta || 0)
+    return (
+      this.value >
+      this.max *
+        (OseDataModelCharacterEncumbrance.encumbranceSteps.quarter / 100)
+    );
+  }
+
+  get color(): string {
```

Comment:
Perhaps a spicy take... I think this should be managed on the template, rather than in the encumbrance class. I'm willing to be wrong, though.